src/model/response.py

```diff
@@ -171,5 +171,5 @@ def full_response(self):
         result['model-id'] = mutated_model_id
     if constants.REQUEST_ID in message:
         result[constants.REQUEST_ID] = message[constants.REQUEST_ID]
-    LOGGER.info('Response for %s is ready in %s sec', message, time.time() - t)
+    logger.info('Response for %s is ready in %s sec', message, time.time() - t)
```

Comment:
Isn't it possible to use the `log_time` context manager?

Reply:
Yes; I didn't really touch other existing log statements except for the variable name, since we're planning to review the statements+levels and I cannot really do that on my own.